src/RestControllers/FHIR/FhirMetaDataRestController.php

```diff
@@ -149,7 +149,7 @@ protected function buildCapabilityStatement()
         $capabilityStatement->setStatus("Not provided");
         $capabilityStatement->addFormat(new FHIRCode("application/json"));
         $resturl = new FHIRUrl();
-        $resturl->setValue("http://" . $_SERVER['SERVER_NAME'] . $gbl::$SITE . $gbl::$ROOT_URL . "/fhir");
+        $resturl->setValue($GLOBALS['site_addr_oath'] . $gbl::$ROOT_URL . "/" . $gbl::$SITE . "/fhir");
```

Comment:
@bradymiller 
Am I correct here? Is `$_SERVER['SERVER_NAME']` one of those server supers that is unreliable in certain environments?
Somehow i'm thinking we need to validate the `$GLOBALS['site_addr_oath']` on app init or somehow report one that is invalid in workflow.

Reply:
yep, would use the $GLOBALS['site_addr_oath'] whenever doing something like this. Then we don't twisted into pretzels when trying to support Server stuff behind multitudes of balancers/proxies in both apache/nginx (and if we ever to figure out a way to support this intrinsically, this will make it centralized, and could do it in globals.php). i think oauth2 basically breaks in OpenEMR if $GLOBALS['site_addr_oath'] is not set correctly.